less/components/popups.less

```diff
@@ -9,7 +9,7 @@
 //--------------------------------------
 
 .popup-shadow {
-    filter: drop-shadow(0px 2px 4px rgba(0, 0, 0, 0.5));
+    box-shadow: 0px 2px 10px rgba(0, 0, 0, 0.5);
```

Comment:
Aw man, this is all we had to do to fix this!?!? I thought it was gonna be a huge shitfight!